.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,43 @@
+name: Build and push image to Docker Hub
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+      -
+        name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      -
+        name: Build and push Strapi
+        uses: docker/build-push-action@v4
+        with:
+          context: "{{defaultContext}}:packages/strapi"
+          file: ./Dockerfile
+          #file: ./Dockerfile.prod
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/restore-strapi:latest
+      -
+        name: Build and push Client
+        uses: docker/build-push-action@v4
+        with:
+          context: "{{defaultContext}}:packages/client"
+          file: ./Dockerfile
+          push: true
```

Comment:
This pushed on PR as well, without out docker labels it will overwrite the image used for production

Reply:
Whooooops :facepalm: thanks!! Will fix